### PR TITLE
MWPW-132352: Added caas e2e tests

### PR DESCRIPTION
**Pull request by zagi25**
Resolves: [MWPW-132352](https://jira.corp.adobe.com/browse/MWPW-132352)

### CODE REVIEW

**Review by Blainegunn — APPROVED** [review]

File: test/e2e/frictionless/features/regression/eventwrapper.feature [hunk 1]
```diff
@@ -50,4 +50,32 @@ Feature: Frictionless Event Wrapper Block
 
   Examples:
       | Verb       | File                |
-      | pdf-to-jpg | test-files/test.pdf |
\ No newline at end of file
+      | pdf-to-jpg | test-files/test.pdf |
+
+  @MWPW-132352 @regression-eventwrapper
```
Comment (TsayAdobe) [inline test/e2e/frictionless/features/regression/eventwrapper.feature:55]: Test case ID is MWPW-132351

**Review by TsayAdobe — CHANGES_REQUESTED** [review]

File: test/e2e/frictionless/step-definitions/dc.steps.js [hunk 2]
```diff
@@ -400,3 +402,32 @@ Then(/^I read expected analytics data with replacements "([^"]*)"$/, async funct
 
   this.page.wikiAnalyticsData = events;
 });
+
```
Comment (JFernandezAdobe) [inline test/e2e/frictionless/step-definitions/dc.steps.js:405]: Can we move the CaaS steps to https://git.corp.adobe.com/wcms/Platform-UI-Adobe?

https://www.npmjs.com/package/@amwp/platform-ui-lib-adobe

Other teams can use these steps too.

**Review by JFernandezAdobe — CHANGES_REQUESTED** [review]

File: test/e2e/frictionless/page-objects/frictionless.page.js [hunk 3]
```diff
@@ -31,6 +31,8 @@ export class FrictionlessPage extends classes(DcGnavPage, PdfWidgetSection) {
       signUp: '[href*="https://auth.services.adobe.com"][href*="signup"]',
       extensionModal: '#chromeext, #edgeext',
       closeExtensionModal: '#chromeext .dialog-close, #edgeext .dialog-close',
+      caas: '[data-path$="/caas/default"]>>nth=0',
```
Comment (JFernandezAdobe) [inline test/e2e/frictionless/page-objects/frictionless.page.js:34]: CaaS are used in other pages too, not only Frictionless pages such as https://www.adobe.com/acrobat/resources/best-pdf-software.html

File: test/e2e/frictionless/step-definitions/dc.steps.js (hunk 2 above)
Comment (zagi25) [reply test/e2e/frictionless/step-definitions/dc.steps.js:405]: If I understood correctly, you want me to port this changes to https://git.corp.adobe.com/wcms/Platform-UI-Adobe, raise the PR there and close this one ?

Comment (TsayAdobe) [reply test/e2e/frictionless/step-definitions/dc.steps.js:405]: It's fine to keep the CaaS steps in this repo for now. We can move them when other teams want to use. In the meantime, we may have to stabilize those steps.

**Review by JFernandezAdobe — APPROVED** [review]

**Review by TsayAdobe — APPROVED** [review]

**Pull request merged**